src/features/counselorSlice.js

```diff
@@ -0,0 +1,83 @@
+import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import axios from "axios";
+
+export const getCounselorInfo = createAsyncThunk(
+  "counselor/getCounselorInfo",
+  async (userId) => {
+    const { data } = await axios.get(
+      `${process.env.REACT_APP_LOCAL_SERVER_URL}/api/users/${userId}`,
+      userId,
+      {
+        withCredentials: true,
+      }
+    );
+
+    return data.data.counselor;
+  }
+);
+
+export const updateAvailableDates = createAsyncThunk(
+  "counselor/updateAvailableDates",
+  async (payload) => {
+    const { data } = await axios.post(
+      `${process.env.REACT_APP_LOCAL_SERVER_URL}/api/users/${payload.userId}/counselor/availableDates`,
+      payload.availableDates,
+      {
+        withCredentials: true,
+      }
+    );
+
+    return data.data.counselor.availableDates;
+  }
+);
+
+const initialState = {
+  isLogging: "",
```

Comment:
```suggestion
  status: "",
```
아래 코드를 봤을때 상태를 나타내는 것 같아서 status라는 변수명이 더 맞는 것 같아요!